Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorSettingsProvider.cs

```diff
@@ -2,16 +2,24 @@
 using System.Collections.Generic;
 using UnityEditor;
 using UnityEngine.UIElements;
+using UnityEditor.UIElements;
 
 namespace UnityEngine.InputSystem.Editor
 {
+    // TODO This editor should react to InputSystem.actions being reassigned from outside
```

Comment:
Is this TODO still valid/

Reply:
Yes, but I will remove the TODOs and cover in Open Issues part of PR instead.

Reply:
Removed